node/src/actors/chain_manager/mining.rs

```diff
@@ -136,6 +145,11 @@ impl ChainManager {
                         Yellow.bold().paint("[Mining]"),
                         Yellow.bold().paint(beacon.checkpoint.to_string())
                     );
+                    let mining_prob =
+                        calculate_mining_probability(&rep_engine, own_pkh, mining_rf, mining_bf);
+                    // Discount the already reached probability
+                    let mining_prob = mining_prob / probability * 100.0;
```

Comment:
This division is no incorrect, right?

```suggestion
```

Reply:
Although I think I would prefer `calculate_mining_probability` to return a number between 0.0 and 1.0, and multiply by 100 here before printing.

Reply:
I was thinking the same before, but it was to not change the other code. Anyways, I will change